src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/RuntimeEnvironmentTest.cs

```diff
@@ -13,9 +13,16 @@ namespace System.Runtime.InteropServices
     public static class RuntimeEnvironmentTest
     {
         [Fact]
-        public static void RuntimeEnvironmentPosTest()
+        public static void RuntimeEnvironmentRuntimeDirectory()
         {
             Assert.True(Directory.Exists(RuntimeEnvironment.GetRuntimeDirectory()));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot,"RuntimeEnvironmentSysVersion not supported on UapAot")]
```

Comment:
Is this something we expect to fix (i.e. should we ActiveIssue rather than SkipOn?)